viamonvifdiscovery/service_test.go

```diff
@@ -0,0 +1,92 @@
+// Package viamonvifdiscovery provides the discovery service that wraps ONVIF integration for the viamrtsp module
+package viamonvifdiscovery
+
+import (
+	"context"
+	"testing"
+
+	"github.com/viam-modules/viamrtsp"
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/services/discovery"
+	"go.viam.com/test"
+)
+
+func TestDiscoveryService(t *testing.T) {
+	cfg := Config{}
+	ctx := context.Background()
+	logger := logging.NewTestLogger(t)
+
+	t.Run("Test Default Service", func(t *testing.T) {
+		testName := "test"
+		resourceCfg := resource.Config{API: discovery.API, Model: Model, Name: testName, ConvertedAttributes: &cfg}
+		dis, err := newDiscovery(ctx, nil, resourceCfg, logger)
+		test.That(t, err, test.ShouldBeNil)
+		test.That(t, dis, test.ShouldNotBeNil)
+		test.That(t, dis.Name().ShortName(), test.ShouldResemble, testName)
+		test.That(t, dis.Close(ctx), test.ShouldBeNil)
+	})
```

Comment:
let's cover more APIs - Name, DoCommand for now.

Reply:
Ignoring do command and name because we do not have any do commands implemented and we do not use rdk’s name function.